src/org/zaproxy/zap/extension/websocket/resources/Messages.properties

```diff
@@ -4,6 +4,8 @@
 
 websocket.api.action.sendTextMessage = Sends the specified message on the channel specified by channelId, \
 if outgoing is 'True' then the message will be sent to the server and if it is 'False' then it will be sent to the client
+websocket.api.action.setBreakTextMessage = Returns a text representation on an intercepted websockets message
```

Comment:
- `on` → `of`?
 - The descriptions are swapped.

Reply:
The second bullet was not addressed.

Reply:
Ooops, missed that - now done :)